pyproject.toml

```diff
@@ -11,6 +11,7 @@ invalid-name,
 fixme,
 too-many-return-statements,
 too-many-ancestors,
+too-many-statements,
```

Comment:
I am not sure about removing this warning. If pylint raises this its usually because a function would benefit from being split between internal functions